LICENSE.txt

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2014 Sean Doyle
+Copyright (c) 2015 Sean Doyle and thoughtbot, inc.
```

Comment:
One is for 2014 and one is for 2015, and they're different. How would you merge this?